sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -418,7 +418,10 @@ export class ApplicationOfferingChangeRequestService {
         id: applicationOfferingChangeId,
         applicationOfferingChangeRequestStatus:
           options?.applicationOfferingChangeRequestStatus,
-        application: { student: { id: options?.studentId } },
+        application: {
+          applicationStatus: ApplicationStatus.Completed,
```

Comment:
An application offering change request can exist only for a completed application, do we need this criteria ?